CHANGELOG.md

```diff
@@ -41,6 +41,12 @@ updated throughout the repository.
 
 ## [Unreleased]
 
+### Changed
+- Using a literal string in macro annotations will now be interpreted as an expression. This makes
```

Comment:
This may cause breakages in others' code, so it'd be nice to call it out explicitly:

```suggestion
- *Breaking Change* Using a literal string in macro annotations will now be interpreted as an expression. This makes
```